README.yaml

```
@@ -317,7 +317,7 @@ usage: |-
       subnet_ids             = module.subnets.public_subnet_ids
 
       local_exec_interpreter = "/bin/bash"
```

We're no longer doing anything very bash-specific. `sh` will have better portability.
```suggestion
      local_exec_interpreter = "/bin/sh"
```